src/arch/xtensa/CMakeLists.txt

```diff
@@ -328,12 +334,21 @@ if(MEU_PATH)
 		USES_TERMINAL
 	)
 else()
+	if(NOT DEFINED ENV{SOF_RIMAGE_KEY})
+		if(NOT DEFINED SOF_RIMAGE_KEY)
```

Comment:
I think it should be just `RIMAGE_KEY`, other variables are named without SOF_ prefix